### ISSUE TITLE
FNX-12602 ⁃ [Bug]Tabs Tray: Tabs are opened only after the second attempt

### ISSUE THREAD

**Issue by AndiAJ** [mozilla-mobile/fenix#11632]
## Prerequisites
Have a couple of open tabs

## Steps to reproduce
1. Open the Tabs Tray
1. Open one by one each tab

### Expected behavior
Each tab is opened on first attempt

### Actual behavior
The tabs are opened only after the second attempt

### Device information
* Android device: 
• Google Pixel 3a (Android 10)
• OnePlus A3 (Android 6.0.1)
* Fenix version: Nightly Build 200616 from 6/16

### Notes
Video
![20200616-201814](https://user-images.githubusercontent.com/51314259/84739688-c60df900-afb4-11ea-8262-85ca47ebc209.gif)

Video
![20200616-201348](https://user-images.githubusercontent.com/51314259/84739645-ab3b8480-afb4-11ea-89f0-667ef28a770e.gif)

**Comment by cadeyrn** [comment]
Duplicate of #11558.
